cpp/modmesh/toggle/RadixTree.hpp

```diff
@@ -310,6 +341,23 @@ class CallProfilerProbe
     CallProfiler & m_profiler;
 }; /* end struct CallProfilerProbe */
 
+class CallProfilerSerializer
```

Comment:
I create ```CallProfilerSerializer``` and move the serialization function in to the ```CallProfilerSerializer```.

Reply:
You should add some comment for the class `CallProfilerSerializer`.  It may be good to point to the google test file.

Reply:
I'm not sure whether the comment is enough to help reading.